src/libs/elektra/kdb.c

```diff
@@ -61,6 +62,19 @@ static void clearAllSync (KeySet * ks)
 	}
 }
 
+static void ksAppendDeepDup (KeySet * ks, const KeySet * toAppend)
```

Comment:
```suggestion
static void ksAppendDup (KeySet * ks, const KeySet * toAppend)
```

I think adding `Dup` is enough to signify it calls `keyDup` compared to `ksAppend` which does nothing.

Reply:
yup